examples/read_bids_datasets.py

```diff
@@ -19,42 +26,57 @@
 """
 # Authors: Adam Li <adam2392@gmail.com>
 #          Richard Höchenberger <richard.hoechenberger@gmail.com>
+#          Alex Rockhill <aprockhill@mailbox.org>
 #
 # License: BSD (3-clause)
 
 ###############################################################################
 # Imports
 # -------
 # We are importing everything we need for this example:
-from mne.datasets import somato
+import os
+import os.path as op
+import openneuro
 
+from mne.datasets import sample
 from mne_bids import BIDSPath, read_raw_bids, print_dir_tree, make_report
 
 ###############################################################################
-# We will be using the `MNE somato data <mne_somato_data_>`_, which
-# is already stored in BIDS format.
-# For more information, you can check out the
-# respective :ref:`example <ex-convert-mne-sample>`.
-
-###############################################################################
-# Download the ``somato`` BIDS dataset
-# ------------------------------------
+# Download a subject's data from an OpenNeuro BIDS dataset
+# --------------------------------------------------------
+#
+# Download the data, storing each in a ``target_dir`` target directory, which,
+# in ``mne-bids`` terminology, is the `root` of each BIDS dataset. This example
+# uses this `EEG dataset <https://openneuro.org/datasets/ds002778>`_ of
+# resting-state recordings of patients with Parkinson's disease.
 #
-# Download the data if it hasn't been downloaded already, and return the path
-# to the download directory. This directory is the so-called `root` of this
-# BIDS dataset.
-bids_root = somato.data_path()
+
+# .. note: If the keyword argument include is left out of
+#          ``openneuro.download``, the whole dataset will be downloaded.
+#          We're just using data from one subject to reduce the time
+#          it takes to run the example.
+
+dataset = 'ds002778'
+subject = 'pd6'
+
+# Download one subject's data from each dataset
+bids_root = op.join(op.dirname(sample.data_path()), dataset)
```

Comment:
Why `op.dirname()`?

Reply:
Thought that was a good place but it could go in the subdirectory too

Reply:
Oh I see. I'd suggest to use `os.pardir` then!